docs/getting-started/getting-started-with-low-code-airflow.md

```diff
@@ -47,7 +47,7 @@ For connecting to Prophecy Managed Airflow, you don't need to provide any other
 
 ### 1.1 Adding AWS Connection
 
-To be able to trigger your Airflow Job, using an S3 File Sensor, you need to have connection from Prophecy Managed Airflow to you S3 account. For this, we need to add an AWS Connection.
+To be able to trigger your Airflow Job, using an S3 File Sensor, you need to have connection from Prophecy Managed Airflow to your S3 bucket. For this, we need to add an AWS Connection.
```

Comment:
Added an [Asana](https://app.asana.com/0/1203813959485797/1205892954315156/f) for a misbehavior